release-1.9/release-1.9.md

```diff
@@ -0,0 +1,89 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
+
+The proposal below is similar in layout previous releases:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- there are a number of North American holidays during Q4
+- same as in 1.8
+  * alpha's should not be cut unless all master-release-blocking tests are passing
+  * release notes for planned features done 2 weeks out from feature freeze
+  * docs PRs deadlines ahead of release
+
+## 1.9 Release Schedule Key Dates
+- Oct 5nd (Weds) coding start
+- Oct 11th (Weds): v1.9.0-alpha.1
+- Oct 25th (Weds): v1.9.0-alpha.2
+- Oct 27th (Fri): Feature Freeze
+- Nov 8th (Weds): v1.9.0-alpha.3
+- Nov 22nd (Weds): v1.9.0-alpha.4
+- Nov 22nd (Weds): Code Freeze 
+- Nov 29th (Weds): v1.9.0-beta.0, v1.10.0-alpha.0
+- Dec 6th (Weds): v1.9.0-beta.release-candidate
+- Dec 8th (Mon): v1.10.0-alpha.1
+- Dec 13th (Weds):  v1.9.0 release!
+
+## 1.9 Details
+
+### Oct 5 - Nov 22
+- 8 week coding period
+- Release 1.9 alphas every 2 weeks
+  - Alphas only cut if [release-master-blocking](https://k8s-testgrid.appspot.com/release-master-blocking) are passing
+- Oct 11th (Weds): v1.9.0-alpha.1
+- Oct 25th (Weds): v1.9.0-alpha.2
+- Oct 27th (Fri): Feature Freeze: all features going into release should
+  have an associated issue in the features repo
+- Oct 31st (Tues): First draft of feature related release notes due
+- Nov 8th (Weds): v1.9.0-alpha.3
+- Nov 15th (Weds): Final draft of feature related release notes due
+- Nov 22nd (Weds): v1.9.0-alpha.4
+
+
+### Nov 22 (Wed) - Dec 6 (Weds)
```

Comment:
Dec 6 - is the first day of KubeCon/CloudNativeCon. Not sure that it's good option to set a deadline for this date.

Reply:
Agreed.

Reply:
Are suggestion later or sooner?